README.md

```diff
@@ -24,27 +24,59 @@
   </tr>
 </table>
 
+
+## Documentation
+
```

Comment:
it was manually generated, yeah. idk. There seems to be an issue with simple navigation of our docs. 

I think this could be solved with an auto-generated website, and this TOC, is pretty much just a site-map.